jointcal/convert_ci_hsc_jointcal_to_exposurecatalogs.py

```diff
@@ -0,0 +1,69 @@
+import lsst.afw.table
+import lsst.afw.image
+import lsst.afw.geom
+import glob
+import os
+import re
+
+
+def convert():
```

Comment:
Some comments in this function would be nice. I follow what's going on except for the block that I discuss in my next comment, but it would have been easier with a little guidance.